cxx-qt-gen/src/generator/naming/qobject.rs

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use crate::generator::naming::CombinedIdent;
+use crate::parser::qobject::ParsedQObject;
+use convert_case::{Case, Casing};
+use quote::format_ident;
+use syn::Ident;
+
+/// Names for parts of a Q_OBJECT
+pub struct QObjectName {
+    /// The name of the stem of the file
+    pub cxx_stem: Ident,
```

Comment:
Needs more thought, should probably come from the module not the QObject at least